tests/Constraints/MinMaxPropertiesTest.php

```diff
@@ -72,7 +72,7 @@ public function getInvalidTests()
         return array(
             array(
                 '{
-                  "value": 1
```

Comment:
because this test was incorrect, the object was found invalid for the wrong reason (because `value` was an integer instead of an object)